test/unit/publishing/publish-by-email.test.js

```diff
@@ -38,9 +38,14 @@ describe('Publish by email', () => {
     expect(mockNotifyClient().prepareUpload).toHaveBeenCalledTimes(1)
   })
 
-  test('should call mockNotifyClient.prepareUpload with FILE_BUFFER and { confirmEmailBeforeDownload: true }', async () => {
+  test('should call mockNotifyClient.prepareUpload with FILE_BUFFER and { confirmEmailBeforeDownload: true, retentionPeriod: config.retentionPeriodInWeeks weeks }', async () => {
     await publishByEmail(EMAIL, FILE_BUFFER, PERSONALISATION)
-    expect(mockNotifyClient().prepareUpload).toHaveBeenCalledWith(FILE_BUFFER, { confirmEmailBeforeDownload: true })
+    expect(mockNotifyClient().prepareUpload).toHaveBeenCalledWith(FILE_BUFFER, { confirmEmailBeforeDownload: true, retentionPeriod: `${config.retentionPeriodInWeeks} weeks` })
+  })
+
+  test('should call mockNotifyClient.prepareUpload with FILE_BUFFER and { confirmEmailBeforeDownload: true, retentionPeriod: 78 weeks }', async () => {
+    await publishByEmail(EMAIL, FILE_BUFFER, PERSONALISATION)
+    expect(mockNotifyClient().prepareUpload).toHaveBeenCalledWith(FILE_BUFFER, { confirmEmailBeforeDownload: true, retentionPeriod: '78 weeks' })
   })
```

Comment:
These tests aren't really testing anything different -- I would go for the latter one.

Ofc more relevant if we were to add tests were we change the config value for `retentionPeriod` to make sure it updates correctly.